src/cmd/ksh93/sh/path.c

```diff
@@ -243,7 +179,7 @@ static pid_t path_xargs(Shell_t *shp,const char *path, char *argv[],char *const
 				saveargs = 0;
 			}
 		}
-		else if(spawn && !sh_isoption(shp,SH_PFSH))
+		else if(spawn /*&& !sh_isoption(shp,SH_PFSH)*/)
```

Comment:
Why comment out this condition rather than just remove it? Other than this (and the same issue below) this LGTM.

Reply:
I left it there just to make it easy to debug on Solaris (if we ever have to). This code is related to forking, so if there are issues around it we can at least take a hint from the commented code.